src/Algebra/Properties/CommutativeSemigroup.agda

```diff
@@ -13,7 +13,6 @@ module Algebra.Properties.CommutativeSemigroup
   where
 
 open CommutativeSemigroup CS
-
```

Comment:
Actually, should keep this blank line: it is an `open` of something specific, which is different than a bunch of `import` of useful functionality.